app-services/secrets/functions/getsecrets.go

```diff
@@ -30,36 +30,36 @@ type getSecretsTestData struct {
 }
 
 var secureSecretsData = []getSecretsTestData{
-	getSecretsTestData{
-		testName:    "Empty path, valid key",
-		path:        "",
-		keys:        []string{"username"},
-		errExpected: false,
-	},
-	getSecretsTestData{
-		testName:    "Fake path, return error",
-		path:        "/fakepath",
-		keys:        []string{"username"},
-		errExpected: true,
-	},
+	// getSecretsTestData{
+	// 	testName:    "Empty path, valid key",
+	// 	path:        "",
+	// 	keys:        []string{"username"},
+	// 	errExpected: false,
+	// },
+	// getSecretsTestData{
+	// 	testName:    "Fake path, return error",
+	// 	path:        "/fakepath",
+	// 	keys:        []string{"username"},
+	// 	errExpected: true,
+	// },
 	getSecretsTestData{
 		testName:    "Valid path, empty keys list, return all secrets",
 		path:        "/mongodb",
 		keys:        []string{},
 		errExpected: false,
 	},
-	getSecretsTestData{
-		testName:    "Valid path, empty string keys, return error",
-		path:        "/mongodb",
-		keys:        []string{"", ""},
-		errExpected: true,
-	},
-	getSecretsTestData{
-		testName:    "Valid path, valid key",
-		path:        "/mongodb",
-		keys:        []string{"username"},
-		errExpected: false,
-	},
+	// getSecretsTestData{
```

Comment:
Why commented out and not just removed?

Reply:
left it in because it was initially for testing some of the functionality for this feature.
could just remove it all together, since really this repo is used to show examples, not for integration tests

Reply:
Yep, remove it.